docs/docs/building-apps/03-protobuf-annotations.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# ProtocolBuffer Annotations
+
+This document explains the various protobuf scalars that have been added to make working with protobuf easier for Cosmos SDK application developers
+
+## Scalars
+
+(cosmos_proto.scalar) = "cosmos.AddressString";
```

Comment:
Just a note to also describe `AddressBytes`, `ValidatorAddressString/Bytes` and `ConsensusAddressString/Bytes`. Basically we want to cover all of what is in the global config and sdk.{Acc/Val/Cons}Address